data/src/main/java/camp/nextstep/edu/github/data/GithubRepositoryImpl.kt

```diff
@@ -1,7 +1,12 @@
 package camp.nextstep.edu.github.data
 
+import camp.nextstep.edu.github.domain.GithubRepo
 import camp.nextstep.edu.github.domain.GithubRepository
 
-internal class GithubRepositoryImpl: GithubRepository {
-
+internal class GithubRepositoryImpl(
+    private val apiService: GithubApiService
+): GithubRepository {
+    override suspend fun getRepo(): List<GithubRepo> {
+        return apiService.repositories().map { it.toDomainModel() }
```

Comment:
```suggestion
        return apiService.repositories().map(GithubRepoResponse::toDomainModel)
```
Input의 타입과 갯수가 동일하면, 메서드 참조를 활용해볼 수 있어요.